Add instructions to download the entire dataset (#4984)
scikit-image/scikit-image: INSTALL.rst

  make system-wide Python changes (installing as root or using sudo),
  you can break the operating system.

Downloading the entire dataset
------------------------------------------------------------------------------

Part of our dataset used in our examples is hosted online and is not installed
by default by the procedures explained above. Data are downloaded once, at the
first call, but this requires an internet connection. If you prefer to download
the entire dataset to be able to work offline, you can run this command:

.. code-block:: sh

    python -c 'from skimage.data import download_all; download_all()'

> 

Reviewer: ```suggestion
or call `download_all()` in your favourite interactive Python environment (IPython, Jupyter notebook, ...)
```

> or call `download_all()` in your favourite interactive Python environment (IPython, Jupyter notebook, ...).

Reviewer: ```suggestion

or call `download_all()` in your favourite interactive Python environment 
(IPython, Jupyter notebook, ...).
```

  make system-wide Python changes (installing as root or using sudo),
  you can break the operating system.

> Downloading the entire dataset

Reviewer 2: "the entire dataset" is a bit unclear without context. How about "Downloading all demo datasets"?

> Downloading the entire dataset

Reviewer: ```suggestion
Downloading all demo datasets
```

Reviewer: ping @sciunto if you accept this suggestion we can merge the PR :-)

Author: I do!

------------------------------------------------------------------------------

> Part of our dataset used in our examples is hosted online and is not installed

Reviewer 3: ```suggestion
Some of the data used in our examples is hosted online and is not installed
```
Nitpick: this minor change sounds better to me
